src/Catty/DataModel/Bricks/Look/NextLookBrick.swift

```diff
@@ -20,37 +20,40 @@
  *  along with this program.  If not, see http://www.gnu.org/licenses/.
  */
 
-#import "PreviousLookBrick.h"
-#import "ObjectTableViewController.h"
-#import "Script.h"
-#import "Pocket_Code-Swift.h"
-
-@implementation PreviousLookBrick
-
-- (kBrickCategoryType)category
-{
-    return kLookBrick;
-}
-
-- (NSString *)brickTitle {
-    return @"";
-}
-
-- (NSString*)pathForLook:(Look*)look
-{
-    return [look pathForScene:self.script.object.scene];
+@objc(NextLookBrick)
+@objcMembers class NextLookBrick: Brick, BrickProtocol {
+    @objc var look: Look?
```

Comment:
The data model of the `NextLookBrick` does not need any `look` property. Please see my comments about the `PreviousLookBrick`, which basically also apply for this Brick.